docs/monitoring.md

```diff
@@ -5,7 +5,7 @@ Firebase Genkit is fully instrumented with
 
 ## Telemetry Configuration
 
-Genkit automatically manages tracing and metrics without requiring explicit configuration. You can enable telemetry exports for Firebase or Google Cloud using their respective plugins and helper functions. Using either plugin poweres the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) that has an AI-idiomatic view of telemetry data.
+Genkit automatically manages tracing and metrics without requiring explicit configuration. You can enable telemetry exports for Firebase or Google Cloud using their respective plugins and helper functions. Using either plugin powers the [Genkit Monitoring dashboard (public preview)](https://firebase.google.com/u/0/project/_/genai_monitoring) that has an AI-idiomatic view of telemetry data.
```

Comment:
`https://firebase.google.com/u/0/project/_/genai_monitoring` is not a working link

Reply:
As an aside - should we just drop the "public preview" label? or is it important to have that in the docs? Seems like something that will go stale.

Reply:
Whoops, fixed the link. Removed public preview too. I think that makes sense given the the Console will still show this message while it is relevant.